homepairs/homepairsUI/src/Screens/Main/Properties/TenantPropertyScreen/TenantPropertyScreen.tsx

```diff
@@ -18,9 +18,9 @@ const mapDispatchToProps: (
     },
   });
 
-const DetailedPropertyScreen = connect(
+const TenantPropertyScreen = connect(
   mapStateToProps,
   // mapDispatchToProps,
 )(TenantPropertyScreenBase);
 
-export default withDarkMode(withNavigation(DetailedPropertyScreen));
\ No newline at end of file
+export default withDarkMode(withNavigation(TenantPropertyScreen));
```

Comment:
Also, we are not using the withDarkMode anymore. If you see any code using it, just make sure to properly remove it and update the wrapped component.